python/nano/src/bigdl/nano/utils/common/schedule.py

```diff
@@ -41,8 +41,10 @@ def get_cpu_info():
         logical_cores = int(subprocess.check_output(
             get_logical_core_args, universal_newlines=True).splitlines()[4].split("=")[1])
         for i in range(logical_cores):
-            for j in range(physical_cores):
-                get_physical_core[i] = j
+            # We don't know how to query which cores are physical cores on Windows,
+            # so we use all logical cores
+            get_physical_core[i] = i
+            get_socket[i] = 0
```

Comment:
This line seems to be repeated with the following two lines. Maybe you can delete the following two lines.